packages/experience/src/components/Button/MfaFactorButton.module.scss

```diff
@@ -0,0 +1,29 @@
+@use '@/scss/underscore' as _;
+
+.mfaFactorButton {
+  padding: _.unit(3) _.unit(4) _.unit(3) _.unit(3);
+  height: unset;
```

Comment:
Why unset height?

Reply:
Since the default button style has a height by default but this factor button's height is determinded by its content.